pkg/alert/config_test.go

```diff
@@ -132,6 +132,10 @@ func TestBuildAlertmanagerConfiguration(t *testing.T) {
 			address: "://user:pass@localhost:9093",
 			err:     true,
 		},
+		{
+			address: "dnssrv+_http._tcp.example.com",
+			err:     true,
+		},
```

Comment:
This test only cover the scheme check, right?

Shouldn't there be one that covers the host error too?